src/components/accessories/patientTriage/PatientTriage.tsx

```diff
@@ -81,39 +100,39 @@ const PatientTriage: FunctionComponent<TProps> = ({
         resetButtonLabel={t("common.discard")}
         shouldResetForm={shouldResetForm}
         resetFormCallback={resetFormCallback}
-        isLoading={isLoading}
+        isLoading={createStatus === "LOADING"}
+      />
+
+      {(createStatus === "FAIL" || deleteStatus === "FAIL") && (
+        <div ref={infoBoxRef}>
+          <InfoBox type="error" message={t("common.somethingwrong")} />
+        </div>
+      )}
+
+      <PatientTriageTable
+        handleDelete={onDelete}
+        shouldUpdateTable={shouldUpdateTable}
       />
-      <div ref={infoBoxRef}>
-        {hasFailed && (
-          <InfoBox
-            type="error"
-            message="Something went wrong, please retry later."
-          />
-        )}
-      </div>
       <ConfirmationDialog
-        isOpen={hasSucceeded}
-        title="Patient Created"
+        isOpen={createStatus === "SUCCESS"}
```

Comment:
See the https://github.com/informatici/openhospital-ui/pull/166#discussion_r685064910